matlab_simulation/04-sensor/scanner_bresenham/example.m

```diff
@@ -25,10 +33,11 @@
 % Generate the belief map
```

Comment:
Just did a quickly check again. I got following error. 
-
Undefined function or variable 'create_a_map'.

Error in example (line 31)
map_real = create_a_map(M,N); % Create a random map with obstacles